torch/distributions/dirichlet.py

```diff
@@ -15,6 +15,12 @@ def _dirichlet_sample_nograd(alpha):
     return probs.clamp_(min=eps, max=1 - eps)
 
 
+def _Dirichlet_backward(x, alpha, grad_output):
```

Comment:
I've exposed this helper for testing.

Reply:
Can you please add that comment in the code?